content/en/docs/collector/deployment/decentralized.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Decentralized
+description:
+  Why and how to send signals to collectors and from there to backends
+weight: 2
+---
+
+The decentralized collector deployment pattern consists of
+applications—[instrumented][instrumentation] with an OpenTelemetry SDK using
+[OpenTelemetry protocol (OTLP)][otlp]—or other collectors (using the OTLP
+exporter) that send telemetry signals to one or more [collectors][collector].
+Each client-side SDK or downstream collector is configured with a collector
+location:
+
+![Decentralized collector deployment concept](../../img/decentralized-sdk.svg)
```

Comment:
For me, this picture doesn’t really capture the decentralized pattern and 1:1 relationship between the application and Collector. You could add 3 apps and 3 Collectors (1-1 connected) to make this visible.